backend/hitas/views/email.py

```diff
@@ -0,0 +1,80 @@
+from typing import Any
+
+from rest_framework import serializers, status
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.serializers import Serializer
+from rest_framework.viewsets import ViewSet
+
+from hitas.services.email import (
+    send_confirmed_max_price_calculation_email,
+    send_regulation_letter_email,
+    send_unconfirmed_max_price_calculation_email,
+)
+from hitas.views.utils import UUIDField
+
+
+class ConfirmedMaxPriceCalculationEmailSerializer(Serializer):
+    calculation_id = UUIDField()
+    template_name = serializers.CharField(max_length=256)
```

Comment:
Should `template_name`s in the Serializers be a `FK`s instead?

Also `EmailTemplateViewSet` could really use a FilterSet :)

Reply:
I think the name is fine since the flow of sending an email is 

1. Search for the template you want to use for the given type of action, e.g. search for confirmed max price calc templates, and pick one (there shouldn't be that many of them, so a filter set shouldn't be required).
2. Send the email, giving the name of the template you selected.